tests/iio_attr.c

```diff
@@ -367,26 +471,27 @@ static void usage(void)
 				options_descriptions[i]);
 		if (i == 3)
 			printf("Optional qualifiers:\n");
-		if (i == 6)
+		if (i == 8)
```

Comment:
the `6` -> `8` change looks a bit fuzzy;
i mean, i could probably find out what it means, but it takes some more digging through the file

Reply:
This was done on the initial implementation - I added a new comment.